cassandane/tiny-tests/JMAPNote/note_set

```diff
@@ -0,0 +1,99 @@
+#!perl
+use Cassandane::Tiny;
+
+sub test_note_set
+    :JMAPExtensions
+{
+    my ($self) = @_;
+
+    my $jmap = $self->{jmap};
+
+    xlog "create note with body but without isHTML";
+    my $res = $jmap->CallMethods([['Note/set',
+                                { create => {
+                                    "1" => {
+                                        title => "HTML",
+                                        body => "<html><head></head><body>Hello World!</body></html>",
+                                    },
+                                  }
+                                },
+                                "R1"]]);
```

Comment:
Nit:

These probably follow our style guide in some way but they make it really hard to read the code.

IMO this is way more readable:
```
my $res = $jmap->CallMethods([[
    'Note/set' => {
        create => {
            "1" => {
                title => "HTML",
                body => "<html><head></head><body>Hello World!</body></html>",
            },
        },
    }, "R1",
]]);
```